book/src/qobject/rustobj_struct.md

```diff
@@ -20,12 +20,12 @@ The RustObj struct allows you to define the following items
 
 ## Invokables
 
-To define a method which is exposed to QML and C++, add a method on the `RustObj` struct and add the attribute `#[invokable]`. The parameters and return type are then matched to the Qt side. Also CXX-Qt automatically adds wrapper code around your invokable to automatically perform any conversion between the [C++ and Rust types](../concepts/types.md).
+To define a method which is exposed to QML and C++, add a method on the `RustObj` struct, with `impl Qt<RustObj>`, and add the attribute `#[invokable]`. The parameters and return type are then matched to the Qt side. Also CXX-Qt adds wrapper code around your invokable to automatically perform any conversion between the [C++ and Rust types](../concepts/types.md).
```

Comment:
Maybe split this up into two paragraphs:
1. What is `impl Qt<RustObj>` (aka, implementation on the C++ QObject that wraps the RustObj - It has access to both C++ and Rust methods).
2. How can I define an invokable on it (#[invokable])

Reply:
Done